components/about_build_modal/about_build_modal.jsx

```diff
@@ -211,7 +211,8 @@ export default class AboutBuildModal extends React.PureComponent {
                     >
                         <FormattedMessage
                             id='about.title'
-                            defaultMessage='About Mattermost'
+                            values={{appTitle: config.SiteName || 'Mattermost'}}
```

Comment:
just to point out this is the same explanation as in the mobile PR, 0/5 if you change it or not